uast/uast_test.go

```diff
@@ -217,3 +217,27 @@ func TestOrderTokens(t *testing.T) {
 		"id3", "Prefix+", "tok_pre_left", "subright_in1", "tok_pre_right", "subright_in2"}
 	require.Equal(expected, result)
 }
+
+func TestCyclomaticComplexity(t *testing.T) {
+	require := require.New(t)
+	n := &Node{InternalType: "empty",
```

Comment:
Its just a parent node, the internalType is just some string for having some field that can be queried for debugging but certainly that one should be something like "module". I will change the code and update the PR.